working/wildcards/feature-specification.md

```diff
@@ -117,11 +117,12 @@ A *local declaration* is any of:
 
     ```dart
     class T<_> {}
+    void genericFunction<_>() {}
 
     takeGenericCallback(<_>() => true);
     ```
 
-A local declaration whose name is `_` does not bind anything to that name. This
+A local declaration whose name is `_` does not bind that name to anything. This
```

Comment:
We're otherwise "binding a name to a value", not "binding a value to a name", so we should probably that direction here as well. (Mnemo: In `'name' -> 42` considered as a notation for binding 'name' to the value 42, the arrow is left-to-right because we will start by finding 'name', and then we'll use the arrow to find 42. The alternate notation `'name' <- 42` seems to allow multiple values to be bound to the same name, which is not intended.)

Reply:
Ah, that mnemonic is very helpful, thank you! I've struggled literally for years with trying to understand whether I should say "bind a name to a value" or "bind a value to a name". Your point that a given name can only have a single value is the key insight I needed.

Reply:
Great, thanks!